storage/s3.go

```diff
@@ -24,14 +24,15 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3/s3iface"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager/s3manageriface"
-
```

Comment:
don't remove this newline :)

Reply:
when we run `go fmt`, it removes this empty line